src/components/common.js

```diff
@@ -505,29 +505,30 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, showDocLink =
         backgroundRepeat: 'no-repeat', backgroundSize: '750px', backgroundPosition: 'right 0 top 0'
       }
     }, [
-      h1({ style: { fontSize: 54 } }, [
-        'Welcome to ',
-        span({ style: { display: isTerra() ? 'block' : 'inline-block' } }, getAppName(isTerra()))
-      ]),
-      div({ style: { margin: '1rem 0', width: 575, ...(bigSubhead ? { fontSize: 20, lineHeight: '28px' } : { fontSize: 16, lineHeight: 1.5 }) } }, [
-        `${getAppName(!isTerra())} is a ${Utils.cond(
-          [isTerra(), () => 'cloud-native platform'],
-          [isFirecloud(), () => 'NCI Cloud Resource project powered by Terra'],
-          [isProjectSingular(), () => 'project funded by Additional Ventures and powered by Terra'],
-          () => 'project powered by Terra'
-        )} for biomedical researchers to `,
-        heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
-        span({ style: { whiteSpace: 'nowrap' } }, ['and', heavyWrapper(' collaborate')]),
-        isProjectSingular() && ' to advance research around single ventricle heart disease',
-        '. ',
-        showDocLink ?
-          h(Link, {
-            style: { textDecoration: 'underline' },
-            href: `https://support.terra.bio/hc/en-us`,
-            ...Utils.newTabLinkProps
-          }, ['Learn more about Terra.']) : null
-      ]),
-      children
+      div({ style: { maxWidth: '50%' } }, [
```

Comment:
This was to limit the cases where text and doc links on the landing page from bleed into the hexagons on the hero image.